### PR TITLE
Create Create-a-search-page-with-verticals-within-the-same-page.md

### PR DESCRIPTION
**Pull request by davemehr**
new scenario article

### CODE REVIEW

**Comment by wobba** [comment]
You need to update index.md to link in the scenario as well. And we prefer if you add the images you use as part of the PR below the assets folder similar to the other md files.

**Comment by davemehr** [comment]
updated the index.md and create a image folder in the assests folder.

File: docs/scenarios/index.md [hunk 1]
```diff
@@ -44,7 +44,11 @@ Storing custom templates as files in a SharePoint site, is great when you want t
 
 When you have your templates in SharePoint, it is easy to setup a way to edit locally on your computer and still get the result in SharePoint almost instantly.
 
+## Scenario 8
 
+[Create a search page with verticals (within the same page)](Create-a-search-page-with-verticals-within-the-same-page.md)
+
+Search Verticals can be used to selectively search, is great when you want to search in predefined result sources, this case will show you, how you config different Search Verticals on the same page.
```
Comment (wobba) [inline docs/scenarios/index.md:51]: Something like this reads better imo:
```
Search verticals can be used to selectively search specific content per vertical. Using the SharePoint provider you can use result sources to limit the content returned, or you can add the required KQL in the web part itself. This sample shows how to set up multiple search verticals on the same page.
```

Comment (davemehr) [reply docs/scenarios/index.md]: I have changed the description. Any other inputs or changes?

**Comment by davemehr** [comment]
Hey @wobba do i have to do anything else for the publication or who will publish the article?

**Comment by wobba** [comment]
@davemehr we'll publish next time we release a v4.

**Pull request merged**